README.md

```diff
@@ -1,2 +1,60 @@
 # Next-Web-Clone
 This is a clone of https://thenextweb.com/ done as a project for Microverse
+
+
+![screenshot](https://github.com/cgcarlosg/Next-Web-Clone/blob/feature/assets/1st-screenshot.png)
+
+
+## Built With
+
+- HTML & CSS
+- Visual Studio, Git & GitHub
+
+
+## Live Demo
+
+[Live Demo Link](https://cgcarlosg.github.io/Next-Web-Clone/.)
+
+
+## Getting Started
+
+Just download the code in your local machine and go to the dowloaded folder, search for the index.html file and open it in your favourite browser.
+
+### Prerequisites
+To run this project, you need a computer or a mobile devise with any operating system and a web browser.
+### Setup
+Just download the code.
```

Comment:
- Optional
- [ ] The person trying to view the project might not be familiar with how to download the code. It would be nice if you can be more descriptive on how to download the code.